bench/icu/optimized/ucnv2022.ll

```diff
@@ -138,50 +138,52 @@ if.then25:                                        ; preds = %if.then23
   br label %if.end219
 
 if.end26:                                         ; preds = %if.then23
-  %tobool30.not = icmp ult i32 %and, 2
-  br i1 %tobool30.not, label %if.end34, label %if.then63
+  %6 = and i32 %3, 1
+  %tobool30.not = icmp eq i32 %6, %and
+  br i1 %tobool30.not, label %if.end34, label %if.then31
```

Comment:
Maybe regression? I'm having a bit of trouble telling as the rest of the codegen looks slightly better.